timescaledb.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "pg", "~> 1.2"
   spec.add_dependency "activerecord"
   spec.add_dependency "activesupport"
+  spec.add_dependency "bytesize"
```

Comment:
I'm a bit skeptical of adding more dependencies for such a small utility. Maybe we should have this code around just to avoid external dependencies for such small detail. How do you feel about it?

Reply:
I'm with you, if we can create a small module to do this, I'm more than happy to remove the dependency, maybe let's try to define the responsibilities of that new module and see if it is something we can include here or keep the dependency if it is too much, what do you think?

Reply:
good point. maybe the dependency will be only in the CLI package ;)

Reply:
Removing it 💪